README.MD

```diff
@@ -141,6 +141,63 @@ The TalendAPI context type requires an _apiUrl_ and an _apiKey_ parameter to con
 
 Any other parameters not listed here can be used as substitution variables in files (when the key is used it is replaced by the value during query execution).
 
+---
+## Actions
```

Comment:
In the code you call this 'Step'. I think that is better name, please use also stepo here in the docs